appinventor/appengine/src/com/google/appinventor/server/encryption/KeyczarEncryptor.java

```diff
@@ -87,7 +87,7 @@ private Crypter getCrypter() throws EncryptionException {
       }
       try {
         // TODO(user): Should we use a different key for development vs deployment?
-        String pathToKeys = rootPath.get() + "keystore";
+        String pathToKeys = rootPath.get() + "/WEB-INF/keystore";
```

Comment:
You cannot have the leading "/", builds fail unless you remove it.

Reply:
Blah. I thought this was relative to the classpath not the file system. I've fixed it.